specification/maps/data-plane/Traffic/preview/1.0/traffic.json

```diff
@@ -718,15 +724,16 @@
               "description": "A single traffic incident, or a cluster of traffic incidents",
               "type": "array",
               "readOnly": true,
+              "x-ms-client-name": "pointsOfInterest",
               "items": {
-                "$ref": "#/definitions/TrafficIncidentPoi"
+                "$ref": "#/definitions/TrafficIncidentPointOfInterest"
               }
             }
           }
         }
       }
     },
-    "TrafficIncidentPoi": {
+    "TrafficIncidentPointOfInterest": {
```

Comment:
When I tested the SDK code, I found two additional properties for this object:
- `sd` : Start date of the incident
- `ed`: Estimated end date of the incident